src/main/java/org/mitre/fhir/authorization/token/Token.java

```diff
@@ -25,4 +33,32 @@ public String getTokenValue() {
     return tokenValue;
   }
 
+  public List<String> getScopes() {
+    return scopes;
+  }
+
+  /**
+   * Gets the list of scopes as a string.
+   * @return a string of each scope separated by a space
+   */
+  public String getScopesString() {
+    String scopesString = "";
+    for (String scope : scopes) {
+      scopesString += scope + " ";
+    }
+
+    scopesString = scopesString.strip();
+
+    return scopesString;
+
+  }
```

Comment:
I think we might want to use `StringBuilder` here.  The compiler might optimize this anyways, but it seems more appropriate regardless.

```suggestion
    StringBuilder scopesString = new StringBuilder();
    for (String scope : scopes) {
      scopesString.append(scope).append(" ");
    }

    return scopesString.toString().strip();
```